### PR TITLE
Revise extending kubernetes

### PR DESCRIPTION
**Pull request by tengqm**
This PR fixes two problems. One was introduced by f265aed794bf30cf5c321941d608a157c9e72a32 which folded the extend-cluster.md content into section index without removing the original content. Another problem is that the PNG images stored on google docs are not accessible by some people from the far end of the planet.

closes: #27440

### CODE REVIEW

File: content/en/docs/concepts/extend-kubernetes/_index.md [hunk 1]
```diff
@@ -103,12 +100,11 @@ This diagram shows the extension points in a Kubernetes system.
 
 If you are unsure where to start, this flowchart can help. Note that some solutions may involve several types of extensions.
 
-
-<img src="https://docs.google.com/drawings/d/e/2PACX-1vRWXNNIVWFDqzDY0CsKZJY3AR8sDeFDXItdc5awYxVH8s0OLherMlEPVUpxPIB1CSUu7GPk7B2fEnzM/pub?w=1440&h=1080">
-
 <!-- image source drawing: https://docs.google.com/drawings/d/1sdviU6lDz4BpnzJNHfNpQrqI9F19QZ07KnhnxVrp2yg/edit -->
+<img src="/docs/concepts/extend-kubernetes/flowchart.png">
```
Comment (sftim) [inline content/en/docs/concepts/extend-kubernetes/_index.md:104]: nit: We should use Markdown image references if we can, and set alt attributes.

**Review by sftim — COMMENTED** [review]
/approve

File: content/en/docs/concepts/extend-kubernetes/_index.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/concepts/extend-kubernetes/_index.md]: revised

File: content/en/docs/concepts/extend-kubernetes/_index.md [hunk 2]
```diff
@@ -80,18 +80,15 @@ and by kubectl.
 Below is a diagram showing how the extension points interact with the
 Kubernetes control plane.
 
-<img src="https://docs.google.com/drawings/d/e/2PACX-1vQBRWyXLVUlQPlp7BvxvV9S1mxyXSM6rAc_cbLANvKlu6kCCf-kGTporTMIeG5GZtUdxXz1xowN7RmL/pub?w=960&h=720">
-
 <!-- image source drawing https://docs.google.com/drawings/d/1muJ7Oxuj_7Gtv7HV9-2zJbOnkQJnjxq-v1ym_kZfB-4/edit?ts=5a01e054 -->
-
+![Extension Points and the Control Plane]("/docs/concepts/extend-kubernetes/control-plane.png")
```
Comment (sftim) [inline content/en/docs/concepts/extend-kubernetes/_index.md:84]:
```suggestion
![Extension Points and the Control Plane](/docs/concepts/extend-kubernetes/control-plane.png)
```
I don't think Markdown expects quotes here.

Comment (tengqm) [reply content/en/docs/concepts/extend-kubernetes/_index.md]: done

**Comment by sftim** [comment]
Preview LGTM

**Comment by sftim** [comment]
LGTM after rebase too!

**Comment by reylejano** [comment]
Preview of updated pages:
https://kubernetes.io/docs/concepts/extend-kubernetes/
https://kubernetes.io/docs/concepts/extend-kubernetes/extend-cluster/

**Comment by reylejano** [comment]
Thank you for making the images on the `extend-kubernetes` page more accessible.
/lgtm

**Pull request merged**